src/runtime/4.8.1/windowsservercore-ltsc2025/Dockerfile

```diff
@@ -0,0 +1,17 @@
+# escape=`
+
+FROM mcr.microsoft.com/windows/servercore/insider:10.0.26304.5000
+
+ENV `
+    # Enable detection of running in a container
+    DOTNET_RUNNING_IN_CONTAINER=true `
+    COMPLUS_RUNNING_IN_CONTAINER=1 `
+    COMPLUS_NGenProtectedProcess_FeatureEnabled=0
```

Comment:
Per https://github.com/microsoft/dotnet-framework-docker/pull/307 is this needed now?

Reply:
Good catch, probably not - I'll give it a try without this.